tasks/configure_gitlab_ce.yml

```diff
@@ -181,6 +181,13 @@
 
 # ---- GitLab installation / upgrade ----
 
+- name: NPM install
+  shell: npm install
+  args:
+    chdir: '{{ gitlab_ce_git_checkout }}'
+  become_user: '{{ gitlab_user }}'
+  when: gitlab_register_ce_checkout is defined and gitlab_register_ce_checkout.changed
```

Comment:
What happens when an older GitLab version is installed, does the `npm install` exit silently when no requirements are provided?